packages/ddd-toolkit/src/command-bus/local-command-bus.ts

```diff
@@ -6,7 +6,7 @@ import { inspect } from 'util';
 export class LocalCommandBus implements ICommandBus {
     private readonly retryMechanism: IRetryMechanism;
 
-    private handlers: { [key: string]: ICommandHandler<ICommand<unknown>> } = {};
+    private handlers: { [key: string]: ICommandHandler<ICommand<unknown, unknown>> } = {};
 
     constructor(
```

Comment:
I would keep the same signature of local-event-bus either change the other

Reply:
@gtoselli my bad the diff hide the rest of the code I missed the usage of the retry. It's perfect sorry.

Reply:
I think I saw an old PR or I was drunk I referred to the LocalCommandBus constructor but probably bec I missed some code.